sys/include/embUnit/AssertImpl.h

```diff
@@ -49,7 +49,7 @@ void assertImplementationCStr(const char *expected,const char *actual, long line
         __typeof__(expected_) ____expected__ = expected_; \
         __typeof__(actual_) ____actual__ = actual_; \
         if (stdimpl_strcmp(____expected__, ____actual__) != 0) { \
-            assertImplementationCStr(____expected__, ____actual__, __LINE__, __FILE__); \
+            assertImplementationCStr(____expected__, ____actual__, __LINE__, RIOT_FILE_RELATIVE); \
```

Comment:
The last version of [embunit is from 2004](http://sourceforge.net/projects/embunit/files/?source=navbar), so an update would be a real surprise.